loopin.go

```diff
@@ -80,6 +81,16 @@ func newLoopInSwap(globalCtx context.Context, cfg *swapConfig,
 	currentHeight int32, request *LoopInRequest) (*loopInInitResult,
 	error) {
 
+	var routeHints [][]zpay32.HopHint
+	var err error
+	if request.Private {
+		routeHints, err = SelectHopHints(globalCtx, cfg.lnd,
```

Comment:
One consideration here is whether we have a `request.LastHop` set. If we're only targetting one peer, it makes sense to only include hops for that peer (minimize privacy leaks).